ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -254,30 +306,13 @@ class TreegraphSeries extends OrganizationSeries {
         }
     }
 
-    public order(node: TreegraphNode, level: number): void {
-        const series = this;
-        // Prevents circular dependencies in the data.
-        if (node.wasVisited) {
-            error(28);
-        } else {
-            level = node.level || level;
-            node.level = level;
-            node.wasVisited = true;
-            node.linksFrom.forEach(function (link: TreegraphPoint): void {
-                if (link.toNode) {
-                    series.order(link.toNode, level + 1);
-                }
-            });
-        }
-    }
-
     /**
      * Calculate `a` and `b` parameters of linear transformation, where
      * `finalPosition = a * calculatedPosition + b`.
      *
      * @return {LayoutModifiers} `a` and `b` parameter for x and y direction.
      */
-    private getLayoutModifiers(): LayoutModifiers {
+    private getLayoutModifier(): LayoutModifiers {
```

Comment:
Is this change of name for some purpose? If yes, change the documentation in comment above :)